screens/Market/MarketScreen.jsx

```diff
@@ -1,15 +1,24 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from "react";
 import { Text, FlatList, TextInput, TouchableOpacity, View, ScrollView, } from 'react-native';
```

Comment:
double quote in row 1 and a single quote in row 2. This doesn't make sense

Reply:
Fixed